### PR TITLE
Fix warnings

### PR DESCRIPTION
**Pull request by jcfr**

### CODE REVIEW

File: Modules/Loadable/Markups/MRML/vtkMRMLMeasurementAngle.cxx [hunk 1]
```diff
@@ -48,42 +48,35 @@ void vtkMRMLMeasurementAngle::Compute()
     }
 
   vtkMRMLMarkupsAngleNode* angleNode = vtkMRMLMarkupsAngleNode::SafeDownCast(this->InputMRMLNode);
```
Comment (jcfr) [inline Modules/Loadable/Markups/MRML/vtkMRMLMeasurementAngle.cxx:50]: @cpinter  When you have a chance, could you review this change ?

File: Modules/Loadable/Markups/VTKWidgets/vtkSlicerROIRepresentation3D.cxx [hunk 2]
```diff
@@ -528,9 +528,6 @@ void vtkSlicerROIRepresentation3D::MarkupsInteractionPipelineROI::GetHandleColor
   double red[4]       = { 1.00, 0.00, 0.00, 1.00 };
   double green[4]     = { 0.00, 1.00, 0.00, 1.00 };
   double blue[4]      = { 0.00, 0.00, 1.00, 1.00 };
-  double orange[4]    = { 1.00, 0.50, 0.00, 1.00 };
```
Comment (jcfr) [inline Modules/Loadable/Markups/VTKWidgets/vtkSlicerROIRepresentation3D.cxx:531]: @Sunderlandkyl Where these colors intended to implement something else ?

Comment (Sunderlandkyl) [reply Modules/Loadable/Markups/VTKWidgets/vtkSlicerROIRepresentation3D.cxx:531]: They can be deleted. I think that they were leftover from when I was experimenting with handle colors.

File: Modules/Loadable/Markups/MRML/vtkMRMLMeasurementAngle.cxx (hunk 1 above)
Comment (jcfr) [reply Modules/Loadable/Markups/MRML/vtkMRMLMeasurementAngle.cxx:50]: cc: @lassoan

**Pull request merged**